storybookjs/storybook
# Typescript: Migrate `manager api` to strict ts

Issue: https://github.com/storybookjs/storybook/issues/22176

## What I did
- Migrated to typescript strict [manager-api]

## How to test
- Navigate to the path using `cd code/lib/manager-api`
- Run `yarn check`
